src/warehouse/identity.js

```diff
@@ -2,7 +2,7 @@ const _ = require('lodash');
 const { InstrumentationError } = require('@rudderstack/integrations-lib');
 const { getVersionedUtils } = require('./util');
 
-const identityEnabledWarehouses = ['snowflake', 'bq'];
+const identityEnabledWarehouses = ['snowflake', 'snowpipe_streaming', 'bq'];
```

Comment:
can we skip this? we're not supporting merge event now rt?

Reply:
Yeah, it might not be required. Removing it.